support-models/src/test/scala/com/gu/support/workers/SendThankYouEmailStateSerialisationSpec.scala

```diff
@@ -78,7 +78,11 @@ object ProductTypeCreatedTestData {
   )
   val paperCreated = SendThankYouEmailPaperState(
     user = User("111222", "email@blah.com", None, "bertha", "smith", Address(None, None, None, None, None, Country.UK)),
-    Paper(fulfilmentOptions = Collection, productOptions = Saturday),
+    Paper(
+      fulfilmentOptions = Collection,
+      productOptions = Saturday,
+      deliveryAgent = Some("A delivery agent with an emoji: 📮"),
```

Comment:
that's brave - zuora historically doesn't handle emojis well at all, often returning SQL errors!.  Not sure if it's changed.

Reply:
Hmm, good point – I wanted to make sure we tested with emojis, but we don’t actually run the IT tests until after merge, do we? I’d better put through an application with an emoji to test it out in the sandbox before I merge.

I believe whether zuora can handle emojis depends on the field (and the emoji: some have low enough codepoints that they work fine, but this isn’t one of them) – maybe it’ll be fine? 😅

Reply:
Sure enough, sending a delivery agent ID with an emoji in it breaks zuora. At the moment it’s just delivery agent IDs we’re storing, which will presumably be numeric – I think we can probably assume those won’t contain emojis, so I’ll remove them from the tests. However, if we add more custom fields (which is a possibility under consideration), we’ll need to remember to exclude characters above U+FFFF and probably raise an alarm in those cases.

Reply:
I’ve removed the emojis